Remora.Commands/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -60,6 +60,31 @@ public static class ServiceCollectionExtensions
             return serviceCollection;
         }
 
+        /// <summary>
+        /// Adds a command module to the available services.
+        /// </summary>
+        /// <param name="serviceCollection">The service collection.</param>
+        /// <param name="commandModule">The command module to register.</param>
+        /// <returns>The service collection with the configured modules.</returns>
+        public static IServiceCollection AddCommandGroup
+        (
+            this IServiceCollection serviceCollection,
+            Type commandModule
```

Comment:
Can you make an overload that accepts an array of Types?

Reply:
Discussed on discord.